pkg/controller/kogitojobsservice/kogitojobsservice_controller.go

```diff
@@ -137,27 +134,9 @@ func (r *ReconcileKogitoJobsService) Reconcile(request reconcile.Request) (resul
 }
 
 const (
-	backOffRetryEnvKey                = "BACKOFF_RETRY"
-	maxIntervalLimitRetryEnvKey       = "MAX_INTERVAL_LIMIT_RETRY"
-	backOffRetryDefaultValue          = 1000
-	maxIntervalLimitRetryDefaultValue = 60000
-	kafkaTopicNameJobsEvents          = "kogito-job-service-job-status-events"
+	kafkaTopicNameJobsEvents = "kogito-job-service-job-status-events"
 )
 
 var kafkaTopics = []string{
 	kafkaTopicNameJobsEvents,
 }
-
-func onDeploymentCreate(cli *kogitocli.Client, deployment *appsv1.Deployment, service appv1alpha1.KogitoService) error {
-	jobService := service.(*appv1alpha1.KogitoJobsService)
-	if jobService.Spec.BackOffRetryMillis <= 0 {
-		jobService.Spec.BackOffRetryMillis = backOffRetryDefaultValue
-	}
-	framework.SetEnvVar(backOffRetryEnvKey, strconv.FormatInt(jobService.Spec.BackOffRetryMillis, 10), &deployment.Spec.Template.Spec.Containers[0])
-	if jobService.Spec.MaxIntervalLimitToRetryMillis <= 0 {
-		jobService.Spec.MaxIntervalLimitToRetryMillis = maxIntervalLimitRetryDefaultValue
-	}
```

Comment:
we don't define default values for those anymore ?

Reply:
I think the default values are already defined at the runtime level

Reply:
@radtriste yes, now operator don't define any values to service configurations. 
Idea with this story is that, defining application properties should be a user resonposiblity. Operator just receive these configuration in key/value pair under `Spec.Config` and append them in `application.properties` into ConfigMap.

Reply:
good for me then :)